easy-install.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+MDBPW="mypass"
+ADMINPW="mypass"
+SITENAME="site1.local"
+BRANCH="develop"
+FRAPPEBRANCH="develop"
+while getopts ":m:a:s:b:f" OPTION; do
+
+  case "$OPTION" in
+    m)
+      MDBPW="$OPTARG"
+      ;;
+    a)
+      ADMINPW="$OPTARG"
+      ;;
+    s)
+      SITENAME="$OPTARG"
+      ;;
+    b)
+      BRANCH="$OPTARG"
+      ;;
+    f)
+      FRAPPEBRANCH="$OPTARG"
+      ;;
+    *)
+      echo "Usage: $(basename $0) [-a admin password] [-m mariadb password] [-s sitename] [-b ERPNext branch] [-f Frappe branch]"
+      exit 1
+      ;;
+  esac
+done
+
+useradd -m -s /usr/bin/bash frappe
+echo 'frappe ALL=(ALL) NOPASSWD: ALL' | sudo tee -a /etc/sudoers
```

Comment:
```suggestion
echo 'frappe ALL=(ALL) NOPASSWD: ALL' | sudo tee -a /etc/sudoers
```

frappe user should have only the permissions it requires. `bench setup sudoers` does just that iirc